requirements/requirements.txt

```diff
@@ -2,11 +2,11 @@
 # see: https://github.com/boto/botocore/commit/e87e7a745fd972815b235a9ee685232745aa94f9
 python-dateutil==2.8.0
 
-boto3==1.*
-holidays==0.9.*
-matplotlib==3.*
-numpy==1.14.*
-pandas>=0.25.0
-pydantic==0.28.*
-tqdm>=4.23.0
-ujson>=1.35
+boto3>=1.0,<2
+holidays>=0.9,<0.10
+matplotlib>=3.0,<4
+numpy>=1.14,<2
+pandas>=0.25,<0.26
+pydantic>=0.28,<2
```

Comment:
The constraint is not `pydantic>=0.28,<1` here because `pydantic` now has versions 1.x which it would be nice to support. Furthermore, I'd like in a subsequent PR to bump `pydantic` to support one of the stable versions.

At the moment, with `pydantic` 1.1, the existing code still works but emits a few warnings that I'll fix in the PR associated with the `pydantic` version bump.